Enterprise-CMCS/macpro-mdct-mfp
# SAR Review and Submit page

### Description

- Creating/updating the verbiage for the Review and Submit page of the SAR for state user and admin (bulk of the PR)
  - admin does not have 'approve' option
  - Admin should see search icon and the text "view" where user sees a pencil and "edit"
- Updating SAR dashboard view for Admin
- Updating and adding tests

### Related ticket(s)


CMDCT-117 & CMDCT-107 & CMDCT-3055

---
### How to test

I've already approved a WP and almost completed a SAR so this is easier to review :)
- Go to https://d3b7qe5ajfnje9.cloudfront.net/
- Log in as `stateuser1` (Texas)
- Go to the SAR Review and Submit page and verify it matches the [Figma](https://www.figma.com/file/oOJMUZerbafZefLQQ33GwI/MFP_Playground?type=design&node-id=2005-95873&mode=design&t=C07lLDCoi3PpQwTg-0)
- Complete the SAR and verify again (alert on review and submit page should be gone). Don't submit yet!
- Login as an admin and verify the [admin view](https://www.figma.com/file/oOJMUZerbafZefLQQ33GwI/MFP_Playground?type=design&node-id=2975-133465&mode=design&t=WFVSN0YrRPLjWhtM-0) for dashboard and Review and Submit
- Login as stateuser1 and submit the SAR
- Verify again for both users, including unlock functionality

Note: The SAR can be endlessly submitted and unlocked, so if another user has already changed it, submit/unlock as needed and re-verify all the phases!

---
### Author checklist


---
